fidesops.toml

```diff
@@ -29,8 +29,6 @@ DRP_JWT_SECRET = "secret"
 LOG_LEVEL = "INFO"
 
 [execution]
-CELERY_BROKER_URL = "redis://:testpassword@redis:6379"
-CELERY_RESULT_BACKEND = "redis://:testpassword@redis:6379"
```

Comment:
these vars are still in the test toml file, but I think we'd want to restore them?